libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -147,14 +147,15 @@ let fedsPlaceholderConfig;
 export const getFedsPlaceholderConfig = ({ useCache = true } = {}) => {
   if (useCache && fedsPlaceholderConfig) return fedsPlaceholderConfig;
 
-  const { locale } = getConfig();
+  const { locale, placeholders } = getConfig();
```

Comment:
Does this tie into another PR? I'm not getting how placeholders end up in the config, it's not obvious from its [definition](https://github.com/adobecom/milo/blob/17e26d4ace3322142375067b9531c87fefc46f3d/libs/utils/utils.js#L207).

Reply:
Didn't check the whole flow, but I would assume this is part of the [personalization logic](https://github.com/adobecom/milo/blob/stage/libs/features/personalization/personalization.js#L526)

Reply:
Could these be just the manifest ones? 🤔 This [line](https://github.com/adobecom/milo/blob/stage/libs/features/personalization/personalization.js#L695) could hint at that. Better to get a quick explanation. Code is straightforward enough, but I'd like to understand that part for my own knowledge.

Reply:
@overmyheadandbody and @mokimo , I actually went through and cleaned up mep related items in config, and placeholders go overlooked.  Since this PR is a blocker, I don't want to hold up merging it for this housekeeping item.  So I made separate ticket for the follow up work.  But I think having the placeholders data nested inside `config.mep` will help clear up this confusion.

Ticket for followup work: https://jira.corp.adobe.com/browse/MWPW-152192